<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationWidget</class>
 <widget class="QWidget" name="CommunicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>695</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Communication</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="csStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="trial_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="trialView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="session_page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeWidget" name="sessionView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="local_page">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTreeWidget" name="localView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="performer_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="performerView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionUpdate">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update data</string>
   </property>
   <property name="toolTip">
    <string>Update data</string>
   </property>
  </action>
  <action name="actionPerformer_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Performer</string>
   </property>
   <property name="toolTip">
    <string>Change view</string>
   </property>
  </action>
  <action name="actionSession_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Session</string>
   </property>
   <property name="toolTip">
    <string>Change view</string>
   </property>
  </action>
  <action name="actionTrial_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Trial</string>
   </property>
   <property name="toolTip">
    <string>Change View</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download server trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLoad_trial">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use trial</string>
   </property>
   <property name="toolTip">
    <string>Use trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbort_download">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Abort download</string>
   </property>
   <property name="toolTip">
    <string>Abort downloading server trial</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLocal_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local</string>
   </property>
   <property name="toolTip">
    <string>Change View</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbort_download</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>abortPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDownload</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>downloadPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_trial</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>loadPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPerformer_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>performerViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSession_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>sessionViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTrial_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>updatePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CommunicationWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLocal_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>localViewPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sessionView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trialView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>789</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>performerView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>CommunicationWidget</receiver>
   <slot>trialContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>861</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>performerViewPressed(bool)</slot>
  <slot>sessionViewPressed(bool)</slot>
  <slot>trialViewPressed(bool)</slot>
  <slot>updatePressed()</slot>
  <slot>downloadPressed()</slot>
  <slot>abortPressed()</slot>
  <slot>contextMenu(QPoint)</slot>
  <slot>loadPressed()</slot>
  <slot>trialContextMenu(QPoint)</slot>
  <slot>addToWorkspace()</slot>
  <slot>removeFromWorkspace()</slot>
  <slot>localViewPressed(bool)</slot>
  <slot>addEMG()</slot>
  <slot>removeEMG()</slot>
  <slot>addGRF()</slot>
  <slot>removeGRF()</slot>
  <slot>addVideo()</slot>
  <slot>removeVideo()</slot>
  <slot>addMocap()</slot>
  <slot>removeMocap()</slot>
  <slot>workspaceContextMenu(QPoint)</slot>
  <slot>workspaceViewPressed(bool)</slot>
  <slot>workspaceTrialViewPressed(bool)</slot>
  <slot>useAllFromWorkspace()</slot>
  <slot>useVideos()</slot>
  <slot>useEmg()</slot>
  <slot>useGrf()</slot>
  <slot>useMocap()</slot>
  <slot>useTrialVideos()</slot>
  <slot>useTrialEmg()</slot>
  <slot>useTrialGrf()</slot>
  <slot>useTrialMocap()</slot>
 </slots>
</ui>
